src/main/java/uk/gov/hmcts/reform/wataskmonitor/config/job/ConfigurationJobConfig.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.hmcts.reform.wataskmonitor.config.job;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ConfigurationProperties(prefix = "job.configuration")
+@Getter
+@Setter
+public class ConfigurationJobConfig {
+    String camundaMaxResults;
+}
```

Comment:
Do we really need a pojo to use this property, can't it be used as @Value

Reply:
Yes, as you mentioned, we can use a @value, but I decided to use this approach to set a framework. We will most likely have to add different contexts/configurations to run different jobs, which seems cleaner than having a few different @values in different jobs. That's the main reason.